packages/account-v2/src/containers/Onfido/OnfidoView.tsx

```diff
@@ -22,34 +20,29 @@ export const OnfidoView = ({
     const { isMobile } = useDevice();
     return (
         <div
-            className={qtMerge(
+            className={twMerge(
                 'relative min-w-[328px] min-h-[500px] m-auto',
                 onfidoCustomClassVariant({ mobile: isMobile })
             )}
         >
             <div className={onfidoInfoMessageVariant({ showStatusMessage })}>
-                <InlineMessage
-                    className='bg-solid-green-1/200'
-                    icon={<IcAlertAnnounce />}
-                    type='filled'
-                    variant='general'
-                >
+                <InlineMessage icon={<IcAlertAnnounce />} type='filled' variant='success'>
```

Comment:
Can u verify the style, seems classname is omitted from the prev implementation

Reply:
In figma all the inlineMessage colors are standardized. Aizad implemented this in UI following figma. All the colors are fixed.